satellite/payments/localpayments/local.go

```diff
@@ -98,27 +102,78 @@ func (*service) GetCustomerPaymentsMethods(ctx context.Context, customerID []byt
 	return []payments.PaymentMethod{defaultPaymentMethod}, nil
 }
 
-// GetPaymentMethod always returns defaultPaymentMethod or error
+// GetPaymentMethod returns defaultPaymentMethod or error otherwise
 func (*service) GetPaymentMethod(ctx context.Context, id []byte) (_ *payments.PaymentMethod, err error) {
 	defer mon.Task()(&ctx)(&err)
 	if string(id) == "0" {
 		return &defaultPaymentMethod, nil
 	}
 
-	return nil, internalPaymentsErr.New("only one payments method exists, with id \"0\"")
+	return nil, localPaymentsErr.New("only one payment method exists, with id \"0\"")
 }
 
-// CreateProjectInvoice creates invoice from provided params
-func (*service) CreateProjectInvoice(ctx context.Context, params payments.CreateProjectInvoiceParams) (_ *payments.Invoice, err error) {
+// CreateProjectInvoice creates invoice from provided params.
+// Returned invoice receives InvoiceStatusPaid by default
+func (s *service) CreateProjectInvoice(ctx context.Context, params payments.CreateProjectInvoiceParams) (_ *payments.Invoice, err error) {
 	defer mon.Task()(&ctx)(&err)
-	// TODO: fill data
-	return &payments.Invoice{}, nil
+
+	var amount int64
+	var lineItems []payments.LineItem
+	var customFields []payments.CustomField
+
+	// line items
+	storage := payments.LineItem{
+		Key:      payments.LineItemStorage,
+		Quantity: int64(params.Storage),
+		Amount:   int64(params.Storage),
+	}
+	egress := payments.LineItem{
+		Key:      payments.LineItemEgress,
+		Quantity: int64(params.Egress),
+		Amount:   int64(params.Egress),
+	}
+	objectCount := payments.LineItem{
+		Key:      payments.LineItemObjectCount,
+		Quantity: int64(params.ObjectCount),
+		Amount:   int64(params.ObjectCount),
+	}
+
+	// custom fields
+	billingPeriod := payments.CustomField{
+		Name:  payments.CustomFieldBillingPeriod,
+		Value: fmt.Sprintf("%s - %s", params.StartDate, params.EndDate),
+	}
+	projectName := payments.CustomField{
+		Name:  payments.CustomFieldProjectName,
+		Value: params.ProjectName,
+	}
+
+	lineItems = append(lineItems, storage, egress, objectCount)
+	customFields = append(customFields, billingPeriod, projectName)
+
+	amount = storage.Amount
+	amount += egress.Amount
+	amount += objectCount.Amount
+
+	inv, err := s.db.CreateInvoice(ctx,
+		payments.Invoice{
+			PaymentMethodID: params.PaymentMethodID,
+			Amount:          amount,
+			Currency:        payments.CurrencyUSD,
+			Status:          payments.InvoiceStatusPaid,
+			LineItems:       lineItems,
```

Comment:
You can use `[]payments.LineItem{storage, egress, objectCount}` directly, the append probably isn't needed.